css/style.css

```diff
@@ -1,3 +1,522 @@
 .page {
     font-family: "Gotham Pro", sans-serif;
   }
+
+.page__header {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  flex-direction: row;
+  margin: 0 0 20px 0;
+  height: 110px;
+}
+
+.header__logo {
+  position: relative;
+  width: 80px;
+  height: 75px;
+  transition: transform 1s linear;
+}
+
+.header__logo:hover {
+  transform: scale(1.1, 1.1);
+}
+
+.header__menu {
+  position: relative;
+  width: 1170px;
+  height: 35px;
+  align-self: center;
+  font-size: 15px;
+  line-height: 11px;
+  text-align: justify;
+  text-transform: uppercase;
+  color: black;
+  font-weight: bold;
+}
+
+.menu__link {
+  color: black;
+  text-decoration: none;
+  align-self: center;
+  transition: text-decoration, font-size, letter-spacing 1s ease;
+}
+
+.menu__link:hover {
+  text-decoration: underline;
+  font-size: 20px;
+  letter-spacing: 2px;
+}
+
+.menu__left {
+  position: absolute;
+  display: flex;
+  width: 500px;
+  height: 30px;
+  justify-content: space-around;
+  align-content: center;
+  left: 13px;
+  bottom: 21px;
+  margin-right: 20px;
+}
+
+.menu__right {
+  position: absolute;
+  display: flex;
+  width: 535px;
+  height: 30px;
+  justify-content: space-around;
+  align-content: center;
+  right: 0px;
+  bottom: 21px;
+}
+
+.menu__search {
+  width: 216px;
+  height: 18px;
+  border-bottom: 1px solid #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)